invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -595,9 +693,17 @@ def draw_ellipse(self, center, width, height, line_width=2, line_colour=(255, 0,
 
         return (xi, yi, xf, yf)
 
-    def draw_rectangle(self, pos, width, height, line_colour=(255, 0, 0, 128),
-        fill_colour=(0, 0, 0, 0), line_width=1, pen_style=wx.PENSTYLE_SOLID,
-        brush_style=wx.BRUSHSTYLE_SOLID):
+    def draw_rectangle(
```

Comment:
pos is tuple of two integers
width is integer
height is integer
line_colour is tuple of 4 integers
fill_colour is tuple of 4 integers
line_width is integer
pen_style is wx.PenStyle
brush_style is wx.BrushStyle

Reply:
Ok